diff --git a/node_modules/instagram-private-api/dist/core/constants.js b/node_modules/instagram-private-api/dist/core/constants.js
index d4485a0..f22b2e3 100644
--- a/node_modules/instagram-private-api/dist/core/constants.js
+++ b/node_modules/instagram-private-api/dist/core/constants.js
@@ -1,8 +1,8 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.WEBHOST = exports.HOST = exports.WEB_HOSTNAME = exports.HOSTNAME = exports.BLOKS_VERSION_ID = exports.FACEBOOK_ORCA_APPLICATION_ID = exports.FACEBOOK_OTA_FIELDS = exports.FACEBOOK_ANALYTICS_APPLICATION_ID = exports.LOGIN_EXPERIMENTS = exports.EXPERIMENTS = exports.SIGNATURE_VERSION = exports.BREADCRUMB_KEY = exports.SIGNATURE_KEY = exports.APP_VERSION_CODE = exports.APP_VERSION = void 0;
-exports.APP_VERSION = '222.0.0.13.114';
-exports.APP_VERSION_CODE = '350696709';
+exports.APP_VERSION = '416.0.0.47.66';
+exports.APP_VERSION_CODE = '382206157';
 exports.SIGNATURE_KEY = '9193488027538fd3450b83b7d05286d4ca9599a0f7eeed90d8c85925698a05dc';
 exports.BREADCRUMB_KEY = 'iN4$aGr0m';
 exports.SIGNATURE_VERSION = '4';
diff --git a/node_modules/instagram-private-api/dist/core/state.js b/node_modules/instagram-private-api/dist/core/state.js
index abc1234..def5678 100644
--- a/node_modules/instagram-private-api/dist/core/state.js
+++ b/node_modules/instagram-private-api/dist/core/state.js
@@ -86,7 +86,7 @@
         return `Instagram ${this.appVersion} Android (${this.deviceString}; ${this.language}; ${this.appVersionCode})`;
     }
     get webUserAgent() {
-        return `Mozilla/5.0 (Linux; Android ${this.devicePayload.android_release}; ${this.devicePayload.model} Build/${this.build}; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/70.0.3538.110 Mobile Safari/537.36 ${this.appUserAgent}`;
+        return `Mozilla/5.0 (Linux; Android ${this.devicePayload.android_release}; ${this.devicePayload.model} Build/${this.build}; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/131.0.6778.135 Mobile Safari/537.36 ${this.appUserAgent}`;
     }
     get devicePayload() {
         const deviceParts = this.deviceString.split(';');
diff --git a/node_modules/instagram-private-api/dist/entities/direct-thread.entity.d.ts b/node_modules/instagram-private-api/dist/entities/direct-thread.entity.d.ts
index 93eb781..2cadf27 100644
--- a/node_modules/instagram-private-api/dist/entities/direct-thread.entity.d.ts
+++ b/node_modules/instagram-private-api/dist/entities/direct-thread.entity.d.ts
@@ -1,11 +1,12 @@
 /// <reference types="node" />
 import { Entity } from '../core/entity';
 import { DirectThreadBroadcastPhotoOptions, DirectThreadBroadcastPhotoStoryOptions, DirectThreadBroadcastReelOptions, DirectThreadBroadcastVideoOptions, DirectThreadBroadcastVideoStoryOptions, DirectThreadBroadcastVoiceOptions } from '../types';
+import { DirectThreadFeedResponseItemsItem } from '../responses';
 export declare class DirectThreadEntity extends Entity {
     threadId: string;
     userIds: string[];
     deleteItem(itemId: string | number): Promise<import("..").StatusResponse>;
-    broadcastText(text: string, skipLinkCheck?: boolean): Promise<import("..").DirectThreadRepositoryBroadcastResponseRootObject | import("..").DirectThreadRepositoryBroadcastResponsePayload>;
+    broadcastText(text: string, reply_to_message?: DirectThreadFeedResponseItemsItem, skipLinkCheck?: boolean): Promise<import("..").DirectThreadRepositoryBroadcastResponseRootObject | import("..").DirectThreadRepositoryBroadcastResponsePayload>;
     broadcastReel(options: DirectThreadBroadcastReelOptions): Promise<import("..").DirectThreadRepositoryBroadcastResponseRootObject | import("..").DirectThreadRepositoryBroadcastResponsePayload>;
     broadcastUserStory(options: DirectThreadBroadcastReelOptions): Promise<import("..").DirectThreadRepositoryBroadcastResponseRootObject | import("..").DirectThreadRepositoryBroadcastResponsePayload>;
     broadcastProfile(id: number | string): Promise<import("..").DirectThreadRepositoryBroadcastResponseRootObject | import("..").DirectThreadRepositoryBroadcastResponsePayload>;
diff --git a/node_modules/instagram-private-api/dist/entities/direct-thread.entity.js b/node_modules/instagram-private-api/dist/entities/direct-thread.entity.js
index 42ec5a0..2a26afe 100644
--- a/node_modules/instagram-private-api/dist/entities/direct-thread.entity.js
+++ b/node_modules/instagram-private-api/dist/entities/direct-thread.entity.js
@@ -17,18 +17,22 @@ class DirectThreadEntity extends entity_1.Entity {
         }
         return this.client.directThread.deleteItem(this.threadId, itemId);
     }
-    async broadcastText(text, skipLinkCheck) {
+    async broadcastText(text, reply_to_message, skipLinkCheck) {
         if (!skipLinkCheck) {
             const urls = text.match(require('url-regex-safe')({ strict: false }));
             if (urls instanceof Array) {
                 return this.broadcastLink(text, urls);
             }
         }
+        let form = { text };
+        if (reply_to_message) {
+            form.replied_to_action_source = 'swipe';
+            form.replied_to_item_id = reply_to_message.item_id;
+            form.replied_to_client_context = reply_to_message.client_context;
+        }
         return await this.broadcast({
             item: 'text',
-            form: {
-                text,
-            },
+            form: form,
         });
     }
     async broadcastReel(options) {
